### DISCUSSION TITLE
Carousel controls not working in canvas but works when previewed in an other page

### DISCUSSION THREAD

**Discussion by patraprerna31** [GrapesJS/grapesjs#4287] [Q&A]
PREV, NEXT of carousel not working on CANVAS.
 I have included these:
 canvas: {
                styles: [
                    'https://surf.service-now.com/styles/scss/sp-bootstrap.scss?portal_id=90a641dddb7a07803d7958a8dc9619e5&v=020f3fd8dbbdf01023b92b69139619a9',
                    'https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300,400,600',
                  'https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css'
                ],
              scripts: [
              'https://code.jquery.com/jquery-3.2.1.slim.min.js',
              'https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js',
              'https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js'
              ]
            },

The same code of carousel works when used on another page having the required style and script links
<img width="1789" alt="Screenshot 2022-04-25 at 10 43 20 PM" src="https://user-images.githubusercontent.com/63239450/165139583-c2640de1-460a-454f-8a3e-cbb84d85d1e3.png">
.

**Comment by ronaldohoch** [comment]
maybe..:
```js
const editor = grapesjs.init({
...
allowScripts: 1,
...
})
```
